### DISCUSSION TITLE
加载完train数据集后加载val时会停止不动，请问是怎么回事？

### DISCUSSION THREAD

**Discussion by xingzhe996** [RL-VIG/LibFewShot#60] [Q&A]
加载完train数据集后加载val时会停止不动，请问是怎么回事？

[10/26/22 11:37:11] INFO     load 2352 train image with 14       trainer.py:372
                             label.                                            
[10/26/22 11:37:34] INFO     load 672 val image with 4 label.    trainer.py:372

程序会一直停在这里，我就是用自己的数据集了而已，用的proto的配置

**Comment by wZuck** [comment]
你可以先尝试在程序中插入打印代码，定位一下问题出在哪一行代码。问题信息不足，我们也没办法复现。

**Comment by xingzhe996** [comment]
我是刚开始学这个，参数设置的含义不懂。
我现在是18类数据，我选了14类做训练（总共2352个样本），4类做验证（总共672个样本），我把RENet参数修改了下，现在可以加载数据了，
shot_num: 1
test_episode: 40
test_epoch: 5
test_query: 15
test_shot: 1
test_way: 3
train_episode: 100
way_num: 3
dataloader_num: 2
batch_size: 16
classifier:
  kwargs:
    feat_dim: 64
    num_classes: 14
这样会出现一个问题，
![图片](https://user-images.githubusercontent.com/83934020/197932559-3b98c4ca-b879-48dc-aed5-3b54228aec52.png)
请问是不是我那里还是设置的不对？

**Reply by wZuck** [reply]
请尝试拉取最新版本的代码，我们在之前的一个commit里修复了该问题。

**Reply by DeeKer2000** [reply]
遇到了同样的问题请问解决了吗，也是卡在加载数据
